tensorflow/tools/ci_build/rel/ubuntu/cpu_arm64_nonpip.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+set -e
+set -x
+
+source tensorflow/tools/ci_build/release/common.sh
+
+# Update bazel
+install_bazelisk
+
+# Set python version string
+python_version=$(python3 -c 'import sys; print("python"+str(sys.version_info.major)+"."+str(sys.version_info.minor))')
+
+# Setup virtual environment
+setup_venv_ubuntu ${python_version}
+
+# Env vars used to avoid interactive elements of the build.
+export HOST_C_COMPILER=(which gcc)
+export HOST_CXX_COMPILER=(which g++)
+export TF_ENABLE_XLA=1
+export TF_DOWNLOAD_CLANG=0
+export TF_SET_ANDROID_WORKSPACE=0
+export TF_NEED_MPI=0
+export TF_NEED_ROCM=0
+export TF_NEED_GCP=0
+export TF_NEED_S3=0
+export TF_NEED_OPENCL_SYCL=0
+export TF_NEED_CUDA=0
+export TF_NEED_HDFS=0
+export TF_NEED_OPENCL=0
+export TF_NEED_JEMALLOC=1
+export TF_NEED_VERBS=0
+export TF_NEED_AWS=0
+export TF_NEED_GDR=0
+export TF_NEED_OPENCL_SYCL=0
+export TF_NEED_COMPUTECPP=0
+export TF_NEED_KAFKA=0
+export TF_NEED_TENSORRT=0
```

Comment:
I don't have familiarity with some of the settings that are used here. Could you let me know where you referenced these from?  I don't see any environment variables being set in the [build script](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/rel/ubuntu/cpu_py310_nonpip.sh) that we used to use for the non-pip tests.

Reply:
they are referenced from here: https://github.com/tensorflow/tensorflow/blob/db84fb33e9c075855685064dedf7923ee05b9eff/tensorflow/tools/ci_build/rel/ubuntu/cpu_arm64_pip.sh#L25